### PR TITLE
Prac 07 feedback

### PR DESCRIPTION
**Pull request by linpyaehtet**
Hey @YY-05 Can you please review the last two files for project management program? I am not sure if I did it correctly since it was a really long exercise. Thank you :)

### CODE REVIEW

File: prac_07/language_file_reader.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+from collections import namedtuple
+
+from programming_language import ProgrammingLanguage
+
+
+def main():
+    """Read file of programming language details, save as objects, display."""
+    languages = []
+    # Open the file for reading
+    in_file = open('languages.csv', 'r')
```
Comment (YY-05) [inline prac_07/language_file_reader.py:17]: with open may be a easy way

File: prac_07/language_file_reader.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+main()
+
+
+def using_csv():
+    """Language file reader version using the csv module."""
+    # First, open the file for reading - note: specify newline
+    # to avoid quoted \n in strings being considered a new record
+    in_file = open('languages.csv', 'r', newline='')
```
Comment (YY-05) [inline prac_07/language_file_reader.py:51]: also

File: prac_07/language_file_reader.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        print(row)
+    in_file.close()
+
+
+# using_csv()
+
+
+def using_namedtuple():
+    """Language file reader version using a named tuple."""
+    in_file = open('languages.csv', 'r', newline='')
```
Comment (YY-05) [inline prac_07/language_file_reader.py:64]: again

File: prac_07/language_file_reader.py
Comment (YY-05) [inline prac_07/language_file_reader.py:1]: Great job!

File: prac_07/programming_language.py
Comment (YY-05) [inline prac_07/programming_language.py:1]: Good job

File: prac_07/project_management.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+DISPLAY_OPTION = 'D'
+FILTER_OPTION = 'F'
+ADD_OPTION = 'A'
+UPDATE_OPTION = 'U'
+QUIT_OPTION = 'Q'
+SAVE_FILE_CHOICE = 'Y'
+
+projects = []
+
+def main():
```
Comment (YY-05) [inline prac_07/project_management.py:29]: docstrings

File: prac_07/project_management.py [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+            print("Invalid menu option, try again")
+        print(MENU)
+        choice = input(">>> ").upper()
+    user_save_choice = input("Would you like to save to projects.txt?(y/n) ").upper()
+    if user_save_choice == SAVE_FILE_CHOICE:
+        save_projects()
+    print(GOODBYE_MESSAGE)
+
+
+def save_projects():
```
Comment (YY-05) [inline prac_07/project_management.py:57]: docstrings

File: prac_07/project_management.py [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+    file_name = input("File name to save projects: ")
+    with open (file_name, "w") as out_file:
+        out_file.write(f"Name\tStart Date\tPriority\tCost Estimate\tCompletion Percentage")
+        for project in projects:
+            out_file.writelines(f"{project.name}\t{project.date}\t{project.priority}\t{project.cost_estimate}\t{project.completion}\n")
+    print(f"Projects saved to {file_name}.")
+
+
+def filter_projects():
```
Comment (YY-05) [inline prac_07/project_management.py:66]: also

**Review by YY-05 — COMMENTED** [review]
Good job, but be careful on docstrings

**Comment by linpyaehtet** [comment]
Thanks :)

**Pull request merged**